cypress/components/menu/menu.cy.tsx

```diff
@@ -937,14 +540,13 @@ context("Testing Menu component", () => {
       }
     );
 
-    it.each([250, 750, 1350])(
+    it.each(["default", "large"] as MenuDividerProps["size"][])(
```

Comment:
why those input values were changed?

Reply:
We can pass only string in size prop ["default", "large"] so I've changed these values but I observed we've covered size prop already so this test is not needed now.